index.html

```diff
@@ -78,6 +90,7 @@ <h1>
             </ul>
           </div>
         </section>
+         </div>
         <!-- Home-page section Ends here -->
 
         <!-- Work-section Starts here -->
```

Comment:
- [ ] Great design for the works section! 🎨 There are still some minor details that you could improve on, please try to stick as much as possible to the Figma template. 🤖 I would suggest incrementing the size of the cards! 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199851618-cec69fb8-44fc-4bc1-beac-570530fdaed7.png) | ![image](https://user-images.githubusercontent.com/77069011/199851653-a478b048-46d7-4bc3-9761-4365bdefcd2a.png) |